crypto/fipsmodule/sha/keccak1600.c

```diff
@@ -1054,11 +347,11 @@ size_t SHA3_Absorb(uint64_t A[SHA3_ROW_SIZE][SHA3_ROW_SIZE], const unsigned char
 
     while (len >= r) {
         for (i = 0; i < w; i++) {
-            uint64_t Ai = (uint64_t)data[0]       | (uint64_t)data[1] << 8  |
-                          (uint64_t)data[2] << 16 | (uint64_t)data[3] << 24 |
-                          (uint64_t)data[4] << 32 | (uint64_t)data[SHA3_ROW_SIZE] << 40 |
-                          (uint64_t)data[6] << 48 | (uint64_t)data[7] << 56;
-            data += 8;
+            uint64_t Ai = (uint64_t)inp[0]       | (uint64_t)inp[1] << 8  |
+                          (uint64_t)inp[2] << 16 | (uint64_t)inp[3] << 24 |
+                          (uint64_t)inp[4] << 32 | (uint64_t)inp[5] << 40 |
```

Comment:
If you reuse the macro `SHA3_ROW_SIZE`, make sure it doesn't replace 5 in `inp[5]` as before :)